src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -126,68 +129,76 @@ function ProductDetails({ product: staleProduct }: Props) {
           />
         </header>
 
-        <ImageGallery images={productImages} />
-
-        <section className="product-details__settings">
-          {skuVariants && (
-            <Selectors
-              slugsMap={skuVariants.slugsMap}
-              availableVariations={skuVariants.availableVariations}
-              activeVariations={skuVariants.activeVariations}
-            />
-          )}
+        <ImageGallery data-fs-product-details-gallery images={productImages} />
 
-          <section className="product-details__values">
-            <div className="product-details__prices">
-              <Price
-                value={listPrice}
-                formatter={useFormattedPrice}
-                testId="list-price"
-                data-value={listPrice}
-                variant="listing"
-                classes="text__legend"
-                SRText="Original price:"
+        <section data-fs-product-details-buying>
```

Comment:
```suggestion
        <section data-fs-product-details-info>
```

How about renaming it to `data-fs-product-details-info`?